content/6_Advanced/parallel_binary_search.mdx

```diff
@@ -0,0 +1,223 @@
+---
+id: parallel-binary-search
+title: 'Parallel Binary Search'
+author: Talha Taki
+prerequisites:
+  - binary-search
+description: It's an amazing technique for doing the binary search for all queries parallely.
+frequency: 0
+---
+
+<FocusProblem problem="sample" />
+
+<Resources>
+	<Resource
+		source="Robert's Blog"
+		title="A SImple Introduction to Parallel Binary Search"
```

Comment:
```suggestion
		title="A Simple Introduction to Parallel Binary Search"
```

Reply:
Changed it! Sorry forgot to take your commit suggestion from here.